modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/SeqTranslate.scala

```diff
@@ -102,6 +102,11 @@ class SeqTranslate(site: Site) {
 
   }
 
+  def instrumentResources(inst: Instrument): Set[Resource] = inst match {
+    case Flamingos2(_) => Set(Resource.Mount, Resource.ScienceFold, Resource.F2)
```

Comment:
That's set when loading a Sequence and set per Step. The `Resource` data types are provisional, we have to come up with a comprehensive list of resources that we'll use for real and carefully pick which resources will be assigned for each kind of Step.